k8s/eks/.env

```diff
@@ -0,0 +1,22 @@
+#### Please fill in the cluster details below: ####
+
+CLUSTER_NAME=default  # This value must be changed or the process will fail!
+REGION=eu-west-3 # Here we declare the AWS region where the cluster and our new EBS will be created
+CNI_COMBINATION=calico_weave # Or change to 'aws_vpc_cni_weave', depending on the desired CNI combination
+
+## NODEGROUP CONFIG ##
+# common
+AVAILABILITY_ZONE=eu-west-3a # For now it must be only one AZ because of the script logic
```

Comment:
We use a hardcoded AZ because, of all the AZs inside our AWS account, we need to find the exact one where our EFS/EBS have been deployed. We need to do this so both the worker nodes and EFS/EBS would be in the same AZ.
If they are not in the same AZ, pairing between kubernetes workloads and AWS resources will fail.

In this iteration it is not possible to do unless we hardcode the exact AZ.
For any future iterations we can add a TODO to enable multiple

Reply:
Why do we even allow configuring AZ then? Shouldn't we just always add an `a` at the end of the region?

Reply:
Because  if you want to use another region other than `eu-west-3` which is set by default, you need to be able to change AZs accordingly.
But I think I understand what do you mean here - we should allow setting up the region variable, and then just add `a` to the region and completely remove the AZ modification variable, right?

Reply:
Sorry, had an afterthought - it doesn't matter if it is `a` or `b` or `c`, it just matters that the AZ is the same for EFS, EBS, and the worker nodes. 
So it is better to keep the AZ variable and the users are able to pick their preferred AZ, as we only care about all resources being scheduled in the same AZ.
Ultimately the users pick which AZ it is going to be.

One example for keeping it that way is someone already having a populated AWS account, and due to the account limitations, maybe their `a` zone is already filled in.

Reply:
If we want to keep it configurable, why can't we just accept the AZ suffix here and use ${REGION}${AZ_SUFFIX}? It'd be less error prone.

Reply:
This was added through `AZ_SUFFIX` variable.